views/movies.ejs

```diff
@@ -0,0 +1,10 @@
+<ul>
+  <% movies.forEach(function(movie) { %>
+    <li><%= movie.Title %>
+    <%= movie.Year %>
+    </li>
+    <img src="<%= movie.Poster %>">
+    <a href="/flick/<%= movie.imdbID %>">Get Deets!</a>
```

Comment:
to get this link to work, create a, app.get for '/flick' and then from there you can pull out the passed in value and access that information from the json file. It's the same as you have done previously, if you need a hand getting this working I can help you go through it.